app.js

```diff
@@ -0,0 +1,125 @@
+
+//--file --validate --links --stats
+
+/*let fs = require('fs')
+function readFile() {
+    let index = process.argv.indexOf("--file")
+    if (index < 0) return console.log("No se encuentra el archivo")
+    let uri = process.argv[index + 1]
+    let fileContent = fs.readFileSync(uri, 'utf8')
+    console.log(process.argv)
+    console.log("index: ", index)
+    console.log("uri: ", uri)
+    console.log("contenido del archivo: ", fileContent)
+    readFile();
+ */
+
+    'use strict';
+var MarkDownIt = require('markdown-it'),
+    md = new MarkDownIt();
+let fs = require('fs');
+let path = require('path');
+let https = require('https');
+let http = require('http');
+const fetch = require('node-fetch');
```

Comment:
Here you could use const not only on this one, but also on the ones before since their value is not gonna change

Reply:
yeah, good point!